tests/integration/tables/os_version.cpp

```diff
@@ -40,7 +41,19 @@ TEST_F(OsVersion, test_sanity) {
       {"install_date", NonEmptyString},
 #endif
   };
+
   validate_rows(data, row_map);
+
+  // Query again with hidden columns too
+  if (isPlatform(PlatformType::TYPE_LINUX)) {
+    data = execute_query(
+        "select *, pid_with_namespace, mount_namespace_id from os_version");
```

Comment:
See my note in https://github.com/osquery/osquery/pull/6414 about making a helper for these :)